test/tests/viewer.js

```diff
@@ -0,0 +1,134 @@
+'use strict';
+/* global Viewer, describe, it, chai, afterEach */
+var expect = chai.expect;
```

Comment:
Does this project make use of the latest version of OBT? `var` is deprecated in favour of `const` & `let`